environments/staging/public.yml

```diff
@@ -127,6 +127,7 @@ localsettings:
   BANK_NAME: "RBS Citizens N.A."
   BANK_SWIFT_CODE: 'CTZIUS33'
   CELERY_PERIODIC_QUEUE: 'celery_null'
+  CELERY_REPEAT_RECORD_DATASOURCE_QUEUE: "repeat_record_datasource_queue"
```

Comment:
we are not doing this for production? I was expecting it to be available there so we can have this on the live environment.

Reply:
I know we offlined but just so others are aware, I'm testing it on staging first and plan to put up a separate PR enabling this on production.